tools/provision/rhel.sh

```diff
@@ -54,8 +72,15 @@ function main_rhel() {
   package clang
   package clang-devel
 
-  set_cc clang
-  set_cxx clang++
+  if [[ $DISTRO = "rhel6" ]]; then
+    # GCC is needed for glib/libstdc++ 4.9+
+    set_cc gcc
+    set_cxx g++
```

Comment:
I am upset about this, but until we can build a LLVM/clang using GCC 4.8+ we have to build our dependencies using the rhscl GCC 4.9 toolchains.